doc/usage/extensions/autosummary.rst

```diff
@@ -303,6 +303,27 @@ The following variables are available in the templates:
 
    .. versionadded:: 1.8.0
 
+.. data:: inherited_methods
+
+   List containing names of only inherited methods.  Only available for
+   classes.
+
+   .. versionadded:: 7.2.7
```

Comment:
I think it'll be probably in 7.3 or 8.x (maybe you could leave a todo and we'll update when the PR is accepted).

Reply:
Have put 7.3.0 for now; Will add a comment below to check this before merging